src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -137,7 +137,6 @@ public static void DigestAuthentication_RoundTrips_Echo()
 
     [Fact]
     [OuterLoop]
-    [ActiveIssue(604, PlatformID.AnyUnix)]
     public static void NtlmAuthentication_RoundTrips_Echo()
```

Comment:
I opened #670 to add a test with explicit credentials.  @mconnew what do you recommend we do with this test now?  Removing ActiveIssue makes it fail.  Are you saying it cannot succeed on Linux using default credentials ever, or is there a new bug here that needs to be opened? Ideally we want all test failures either deactivated with [ActiveIssue] or left failing for further investigation.